src/frontend/src/lib/components/navigation/NavigationMenu.svelte

```diff
@@ -11,26 +12,22 @@
 	import {
 		isRouteDappExplorer,
 		isRouteSettings,
+		isRouteTokens,
 		isRouteTransactions,
 		networkParam
 	} from '$lib/utils/nav.utils.js';
 
-	let route: 'transactions' | 'tokens' | 'settings' | 'explore' = 'tokens';
-	$: route = isRouteSettings($page)
-		? 'settings'
-		: isRouteDappExplorer($page)
-			? 'explore'
-			: isRouteTransactions($page)
-				? 'transactions'
-				: 'tokens';
+	// If we pass $page directly in the HTML, we get a type error
```

Comment:
What error? Maybe worth documenting it?

Reply:
It is a very strange warning: I searched around but I couldn't find something related:

![Screenshot 2024-11-06 at 09 55 27](https://github.com/user-attachments/assets/0bae553f-430b-4548-a71f-112553b82eb9)

But I included in the comment: have a look please